mms/tests/unit_tests/test_worker_service.py

```diff
@@ -40,7 +40,7 @@ def test_with_nil_request(self, service):
             service.retrieve_data_for_inference(None)
 
     def test_valid_req(self, service):
-        input_batch, req_to_id_map = service.retrieve_data_for_inference(self.data)
+        headers, input_batch, req_to_id_map = service.retrieve_data_for_inference(self.data)
```

Comment:
Just curious, is this the only unit test that is affected by this change ?

Reply:
Yes, this is seems to be the only one affected. I think I will update this test though, to test for headers, even though this is an optional field.